.gitignore

```diff
@@ -1 +1,7 @@
-target/*
\ No newline at end of file
+<<<<<<< HEAD
+target/
+
+
+=======
+target/*
+>>>>>>> b6cc154f3f1d837608ee965f1b871f3f78edeaec
```

Comment:
I am not sure how to change this because when I look at my git ignore file, I see lines 2 through 7, not line one which is what I think needs to be deleted so there are no conflicts

Reply:
You should delete the ">>>" and "<<<<" and "====", which are nonsense added by git. And also delete one of the redundant `target/` lines.